### PR TITLE
Fix `exec: already started` error in uninstall on Linux

### PR DESCRIPTION
**Pull request by RebeccaMahany**
Keeps us from attempting to run the systemctl disable command twice.

### CODE REVIEW

**Review by directionless — DISMISSED** [review]

File: cmd/launcher/uninstall_linux.go [hunk 1]
```diff
@@ -22,12 +22,8 @@ func removeLauncher(ctx context.Context, identifier string) error {
...
 	// Stop and disable launcher service
 	cmd := exec.CommandContext(ctx, "systemctl", []string{"disable", "--now", serviceName}...)
-	if out, err := cmd.Output(); err != nil {
-		fmt.Printf("error occurred while stopping/disabling launcher service, systemctl output %s: err: %s\n", out, err)
-		return err
-	}
-
-	if err := cmd.Run(); err != nil {
+	if out, err := cmd.CombinedOutput(); err != nil {
+		fmt.Printf("error occurred while stopping/disabling launcher service, systemctl output %s: err: %s\n", string(out), err)
```
Comment (directionless) [inline cmd/launcher/uninstall_linux.go:26]:
```suggestion
	if out, err := cmd.CombinedOutput(); err != nil {
		// Don't exit. Log and move on to the next uninstall command
		fmt.Printf("error occurred while stopping/disabling launcher service, systemctl output %s: err: %s\n", string(out), err)
```

Comment (RebeccaMahany) [reply cmd/launcher/uninstall_linux.go]: Wait sorry, applied before I checked the full diff -- do you want me to also remove the `return err` below this, then?

Comment (directionless) [reply cmd/launcher/uninstall_linux.go]: I mean, I also approved it before I added the suggestion. 😆 

Reading the file, I think all the `return err` should probably be removed. The ones around `dpkg` and `rpm` feel suspect.

Comment (RebeccaMahany) [reply cmd/launcher/uninstall_linux.go]: Done!

**Review by directionless — APPROVED** [review]

**Pull request merged**